.github/workflows/cypress-workflow.yml

```diff
@@ -106,7 +106,7 @@ jobs:
       - name: Bootstrap plugin/OpenSearch-Dashboards
         run: |
           cd OpenSearch-Dashboards/plugins/security-analytics-dashboards-plugin
-          yarn osd bootstrap
+          yarn osd bootstrap --single-version=loose
```

Comment:
This is necessary after core added a dependency on cypress 9.5.4. See https://github.com/opensearch-project/OpenSearch-Dashboards/issues/5561 for more context